### ISSUE TITLE
i have problem with fud backdoor avoid v1.2 dont know how to listen

### ISSUE THREAD

**Issue by medoomem** [Screetsec/TheFatRat#732]
i made a succesfull backdoor using the third option of fatrat
[03]  Create Fud Backdoor with Avoid v1.2 
but idk how to listen to the payload because i dont know which payload it uses since it does not option me on which payload i wanna use

so i cant open msfconsole exploit/multi/handler

because i dont know which payload should be set

help?

**Comment by Hamlet3211** [comment]
Did you have problems with the searchsploit?

**Comment by medoomem** [comment]
what do i do in searchsploit? you see im new to fatrat :)

**Comment by medoomem** [comment]
when i use fatrat i always usit to make backdoors with the first option which is msfvenom
and i set the payload in exploit multi handler

should i search in searchsploit for something?

**Comment by Hamlet3211** [comment]
No, I was just wondering if you had an issue with it in the past. Currently the only thing I'm stuck on.

**Comment by medoomem** [comment]
oh i have no problem with searchsploit

**Comment by medoomem** [comment]
whats your problem

**Comment by Hamlet3211** [comment]
When I run setup.sh it s the only thing it can’t find even though I’ve
installed it

**Comment by kingeggobandit** [comment]
Have you been able to make a good backdoor amd listen yet? 

Are you also able to reconnect to that pc afterwords with that backdoor

Will you have persistence

**Comment by maksoftsolutions** [comment]
how listen ps1 backdoor
